editor/plugins/org.fusesource.ide.wsdl2rest.ui/build.properties

```diff
@@ -15,7 +15,6 @@ bin.includes = META-INF/,\
                libs/commons-beanutils.jar,\
                libs/commons-collections.jar,\
                libs/commons-digester.jar,\
-               libs/commons-lang.jar,\
```

Comment:
I think commons-lang needs to be kept

![image](https://user-images.githubusercontent.com/1105127/57068149-28cd0400-6cd1-11e9-9e7b-26f6f8be8df1.png)